doc-content/enterprise-only/installation/jws-datasource-proc.adoc

```diff
@@ -1,5 +1,5 @@
 [id='_setting_up_transaction_manager']
-==== Setting up Transaction Manager for Red Hat JBoss Web Server 2.1 (Tomcat 7)
+==== Setting up Transaction Manager for {JWS} 2.1 (Tomcat 7)
```

Comment:
Judging from the version used here `2.1` Why do we still have this in documentation? Am I missing something?

Reply:
It's not in the doc. Some legacy files were globally updated with the new variable. If in doubt, refer to the rendered drafts.

Reply:
Do we need to keep legacy things here? Can't we just remove them?